spec/project/changelog_spec.rb

```diff
@@ -4,7 +4,7 @@
   it 'has link definitions for all implicit links' do
     implicit_link_names = changelog.scan(/\[([^\]]+)\]\[\]/).flatten.uniq
     implicit_link_names.each do |name|
-      expect(changelog).to include("[#{name}]: http")
+      expect(changelog).to include("[#{name}][]")
```

Comment:
This should no longer be necessary once you add yourself to the end of the CHANGELOG file.